### PR TITLE
Combines contact choice and contact details into single political contact form

### PR DESCRIPTION
**Pull request by logicgraphy**
- Removes contact choice form and container
- Removes contact details form and container
- Adds political contact container form as a controller for contact choice and details.

### CODE REVIEW

**Comment by baccigalupi** [comment]
Hey, noticed the CodeClimate shows duplication on the email etc. Do we need both forms of the contact for what is visible.

File: client/helpers/data-present.js [hunk 1]
```diff
@@ -59,6 +53,18 @@ const previousNamesInfo = (props) => {
   );
 }
 
+const politicalContact = (props) => {
+  return props && (
+    value(props.emailAddress) || value(props.phoneNumber) || value(props.shouldContact)
+  );
+}
+
```
Comment (baccigalupi) [inline client/helpers/data-present.js]: These look like the same thing. Probably we want one function named well enough that it makes sense in each context!

File: client/presentations/summary/political-contact.jsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  if(shouldContact !== 'Yes') {
+    if(shouldContact === 'Skip Question') { shouldContact = 'No Answer'; }
+    return (
+      <div className='summary-section'>
+        <p> Should Contact: {shouldContact} </p>
+      </div>
+    );
+  }
+
+  //if (!dataPresent.contactDetails(props.politicalContact)) { return null; }
```
Comment (baccigalupi) [inline client/presentations/summary/political-contact.jsx]: Oh this is that null return I was talking about earlier! Anyway, let's get this comment out if we don't need it.

File: client/actions/index.js [hunk 3]
```diff
@@ -32,16 +31,15 @@ export const TYPES = {
   UPDATE_POLITICAL_PARTY_CHOOSE:          'UPDATE_POLITICAL_PARTY_CHOOSE',
   UPDATE_POLITICAL_PARTY_PREFERENCE:      'UPDATE_POLITICAL_PARTY_PREFERENCE',
   UPDATE_BALLOT_LANGUAGE:                 'UPDATE_BALLOT_LANGUAGE',
-  UPDATE_CONTACT_CHOICE:                  'UPDATE_CONTACT_CHOICE',
-  UPDATE_OPT_OUT:                         'UPDATE_OPT_OUT'
+  UPDATE_OPT_OUT:                         'UPDATE_OPT_OUT',
```
[the remark this answers is not in the document]
Comment (logicgraphy) [reply client/actions/index.js]: We combined action and reducer for 'contact choice' and 'contact details' into one called 'political-contact'

**Pull request merged**